src/doom/g_game.c

```diff
@@ -94,6 +94,10 @@ void	G_DoVictory (void);
 void	G_DoWorldDone (void); 
 void	G_DoSaveGame (void); 
  
+// [crispy] Write level statistics upon exit
```

Comment:
I don't think we need these declarations here.

Reply:
Good point. I'm so used to flying blind in the doom codebase and C that I do some things without thinking if it makes sense. It would be the same for heretic as well right? I'll fix it when I add total time tracking there as well.